event-log/src/main/scala/io/renku/eventlog/subscriptions/newEvent/ProjectPrioritisation.scala

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package io.renku.eventlog.subscriptions
+package io.renku.eventlog.subscriptions.newEvent
```

Comment:
What do you think if the package name would be smth like `newandrecoverable` or better?

Reply:
Ok good to know :D

Reply:
We went for unprocessed. As NewAndRecoverabelEventsFetcherImpl was getting long